calculator_data.yml

```diff
@@ -0,0 +1,70 @@
+---  # document start
+
+calendar_id : vakt@jodiskmuseumoslo.no
+
+email_sender : mejlzatestiranje69@gmail.com
```

Comment:
I don't see the reason why this s**t exists in the first place.

If one already has to log in into a Google account, to access the calendar and Google drive, why can't you reuse his credentials to send the mail?